cmd/selfsync.go

```diff
@@ -0,0 +1,234 @@
+package cmd
```

Comment:
keep the cmd package very small. it should just gather the CLI options and call a function in the ytsync package. `lbry.go` is shared across many different projects. keep your changes in `ytsync` package as much as possible.